src/Lumi/Components/Select/SelectBackend.purs

```diff
@@ -130,7 +130,9 @@ selectBackend = make component
       when (isOpen /= self.state.isOpen) do
         send self if isOpen then CloseSelect else OpenSelect
 
-    send self@{ props, state } action =
+    send self action = do
+      props <- readProps self
+      state <- readState self
```

Comment:
This send-self pattern has race conditions.. it all seems to be working at the moment, but I would rather refactor the whole component at some point. Anyway, there are cases where `self.state` was stale, and this function is already an `Effect` so it now uses `readState self` instead to ensure the most recent state possible. This still isn't as good as using the state available within `self.setState \state -> ...` but that api doesn't allow conditional effects.. so yeah, this fixes the current bug but it'd be better to rewrite this without `React.Basic.Classic`.